### DISCUSSION TITLE
How to run Doris broker service using helm chart

### DISCUSSION THREAD

**Discussion by nikola-arcadis** [apache/doris#40206] [A - General / Q&A]
I'm using SelectDB Doris helm chart and was able to deploy 1 FE and 3 BE, but when I enable Broker then broker pod keeps printing this log
`not have leader wait fe cluster select a master, sleep 2s...`
And the pod keeps being unhealthy and not running any broker loads.

**Comment by ixzc** [comment]
Seems to be the reason it can't access the fe master. we need the broker's pod logs and your helm values.yaml configuration. you can use s3 load, it's more easier and needn't to add a broker.

**Comment by nikola-arcadis** [comment]
Here is my helm chart config:

`dorisCluster:
        enabledBroker: true
      beSpec:
        replicas: 3
        resource:
          requests:
            cpu: 300m
            memory: 768Mi
        persistentVolumeClaim:
          dataPersistentVolume:
            storage: 50Gi
          logsPersistentVolume:
            storage: 20Gi
      feSpec:
        replicas: 1
        persistentVolumeClaim:
          metaPersistentVolume:
            storage: 30Gi
          logsPersistentVolume:
            storage: 20Gi
        resource:
          requests:
            cpu: 300m
            memory: 768M
      brokerSpec:
        replicas: 1
        persistentVolumeClaim:
          metaPersistentVolume:
            storage: 30Gi
          logsPersistentVolume:
            storage: 20Gi
        resource:
          requests:
            cpu: 300m
            memory: 768M
    `
So, what do you recommend using broker for? Will s3 load scale well?
Is it possible to define my own broker implementation easily? Like I want to fetch data from an HTTP API, for example?
